src/Operation/Aggregate.php

```diff
@@ -302,6 +309,10 @@ private function createCommand(Server $server)
             $cmd['collation'] = (object) $this->options['collation'];
         }
 
+        if (isset($this->options['explain'])) {
```

Comment:
This can be rolled into the `foreach (['comment', 'maxTimeMS'] as $option)` loop above, since it uses the basic isset/assign behavior with no casting or other logic.